cmd/ndm_daemonset/probe/eventhandler.go

```diff
@@ -213,14 +197,27 @@ func generateUUID(bd blockdevice.BlockDevice) (string, bool) {
 	return uuid, ok
 }
 
-func (pe *ProbeEvent) addBlockDevice(bd blockdevice.BlockDevice) error {
-
-	bdAPIList, err := pe.Controller.ListBlockDeviceResource(true)
-	if err != nil {
-		klog.Error(err)
-		return err
+// generate old UUID, returns true if the UUID has used path or hostname for generation.
```

Comment:
As we are refactoring this huge file into smaller files, these helper function can be moved to their own file - probably uuidHelpers.go ?